releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,6 +1,37 @@
 ## Microsoft.Extensions.Configuration.AzureAppConfiguration
 ### [Package (NuGet)](https://www.nuget.org/packages/Microsoft.Extensions.Configuration.AzureAppConfiguration)
 
+### 6.0.0 - March 28, 2023
+### Breaking Changes:
+* Removed `IConfigurationRefresher.SetDirty` API in favor of `IConfigurationRefresher.ProcessPushNotification` API for push-model based configuration refresh. Unlike the `SetDirty` method, the `ProcessPushNotification` method guarantees that all configuration changes up to the triggering event are loaded in the following configuration refresh. For more details on the `ProcessPushNotification` API, refer to [this tutorial](https://docs.microsoft.com/en-us/azure/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh?tabs=windowscommandprompt). [#357](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/357)
+* Removed .NET 5 as a target framework as .NET 5 is out of support. [#391](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/391)
+* Feature Management V2 schema support, which was introduced in 5.2.0-preview release, has been removed from this stable release. [#315](https://github.com/Azure/AppConfiguration-DotnetProvider/issues/315)
+* Removed IConfigurationRefresher.LoggerFactory API, but refresh logs are still available through standard ASP.NET Core logging if services.AddAzureAppConfiguration() is invoked in your ConfigureServices method. [#367](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/367)
+
+* This is the first stable release of the following API introduced in 5.3.0-preview release:
```

Comment:
These two new APIs are not breaking changes. They are enhancements.

Reply:
That's true. I think we can move these APIs to the enhancements section, but we should call out a breaking change that was introduced while adding geo replication support. 
We now check to make sure that push notification is processed only for registered endpoints ([code](https://github.com/Azure/AppConfiguration-DotnetProvider/blob/cd590f32b4efdbca0b55a5287b6609c7166871d7/src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs#L478)). Earlier, we were invalidating cache without validating the push notification.